.planning/codebase/STRUCTURE.md

```
@@ -0,0 +1,202 @@
+│   │   └── fetcher.go          # Fetcher interface
+│   └── scanner/
+│       └── scanner.go          # Scanner, ParseInput, GetSong{Multi,Text,Dir}, AssertInput
+├── .claude/                    # Claude Code configuration
+│   ├── commands/               # Custom slash commands
+│   ├── settings.json
+│   └── settings.local.json     # (gitignored)
+├── .githooks/
+│   └── pre-commit              # Pre-commit quality checks
+├── .github/
+│   ├── workflows/
+│   │   ├── ci.yml
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)